src/lib/server/session.cpp

```diff
@@ -25,13 +25,15 @@ void Session::run() {
   while (!_terminate_session) {
     try {
       _handle_request();
+      // clang-format off
```

Comment:
What would clang-format do here?

Reply:
it removes the line breaks before/after `return` inside `catch`. For an example. see https://github.com/hyrise/hyrise/pull/2487#discussion_r931931055.